go/libkb/env.go

```diff
@@ -1763,9 +1763,12 @@ func (e *Env) ModelessWantsSystemd() bool {
 		os.Getenv("KEYBASE_SYSTEMD") != "0")
 }
 
-func (e *Env) DarwinForceSecretStoreFile() bool {
-	return (e.GetRunMode() == DevelRunMode &&
-		os.Getenv("KEYBASE_SECRET_STORE_FILE") == "1")
+func (e *Env) ForceSecretStoreFile() bool {
+	// By default use system-provided secret store (like MacOS Keychain), but
+	// allow users to fall back to file-based store for testing and debugging.
+	return e.GetBool(false,
+		func() (bool, bool) { return e.getEnvBool("KEYBASE_SECRET_STORE_FILE") },
```

Comment:
Can we put this in config as well? So if a user has an issue with their keychain we can tell them to just flip the config flag on and not worry about env vars

Reply:
fair point. I was going to not do a config variable not to encourage people to use files if they can avoid it, but it would be good to have one